src/containers/main/NewProjectGenerator.js

```diff
@@ -0,0 +1,80 @@
+import React, {Component} from "react";
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {createNewProject} from "../../actions"
+
+class NewProjectGenerator extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      company: '',
+      project: '',
+      suite: '',
+      domain: '',
+      tests: null
+    }
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentDidMount() {
+    document.getElementsByClassName("test-container")[0].style.overflow = "hidden";
```

Comment:
Can this be set up as react state functionality?

Reply:
Done.